contracts/staking/TokenStaking.sol

```diff
@@ -697,7 +714,7 @@ contract TokenStaking is Ownable, IStaking {
             "Too much to unstake"
         );
         operatorStruct.nuInTStake -= amount;
-
+        decreaseStakeCheckpoint(operator, amount);
```

Comment:
Nitpick but we could keep the same order of operations of emitting event, updating staked amount, and decreasing checkpoint in `unstake`, `unstakeKeep`, and `unstakeNu`.